### DISCUSSION TITLE
代码无法高亮

### DISCUSSION THREAD

**Discussion by drongsidj** [izhangzhihao/intellij-rainbow-brackets#2701] [Q&A] [answered]
我的代码中变量名、类名等无法高亮显示，请问如何设置，我的idea版本是
Build #IU-233.13135.103, built on December 20, 2023
![Snipaste_2023-12-30_18-06-37](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/53135879/adfe75a0-afc4-4a4f-bab2-be17d6a55eb0)

**Comment by izhangzhihao** [comment, marked as answer]
Might duplicate with https://github.com/izhangzhihao/intellij-rainbow-brackets/issues/2686

**Comment by izhangzhihao** [comment]
Closed for no update.
